utils/try-catch.js

```diff
@@ -0,0 +1,44 @@
+const CORS_ERROR_HEADER = {
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Credentials": true
+};
+
+async function tryCatch ( inputFunction ){
```

Comment:
ah just revert all the tryCatch stuff, it doesn't look good
we'll keep the CORS

Reply:
ok. i have reverted the changes. also renamed CORS_ERROR_HEADER to CORS_POLICY_HEADER for more clarity. let me know if it is ok.

Reply:
hi rj, i have rebased to the latest version of master already. tests are passing now.

Reply:
it doesn't look rebased to me, did you do a git rebase?

Reply:
sorry nevermind i saw wrongly